### PR TITLE
tada68: add shalzz's layout

### PR DESCRIPTION
**Pull request by shalzz**
Made some improvements (atleast for me) to the already great unix layout for tada68.

### CODE REVIEW

**Comment by shalzz** [comment]
@noroadsleft done!

File: keyboards/tada68/keymaps/shalzz/rules.mk [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# Build Options
+#   change to "no" to disable the options, or define them in the Makefile in
+#   the appropriate keymap folder that will get included automatically
+#
+BOOTMAGIC_ENABLE = no       # Virtual DIP switch configuration(+1000)
```
Comment (noroadsleft) [inline keyboards/tada68/keymaps/shalzz/rules.mk]: For a keymap-level `rules.mk`, you only need to define the rules that are different from those set in the keyboard's `rules.mk` file.

For you, that means:

```
MOUSEKEY_ENABLE = yes       # Mouse keys(+4700)
CONSOLE_ENABLE = no         # Console for debug(+400)
COMMAND_ENABLE = no         # Commands for debug and configuration
NKRO_ENABLE = yes           # Nkey Rollover - if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
BACKLIGHT_ENABLE = yes      # Enable keyboard backlight functionality
AUDIO_ENABLE = no           # Audio output on port C6
RGBLIGHT_ENABLE = no        # Enable WS2812 RGB underlight.  Do not enable this with audio at the same time.
SLEEP_LED_ENABLE = yes      # Breathing sleep LED during USB suspend
```

**Comment by drashna** [comment]
Awesome, thanks!

**Comment by shalzz** [comment]
Thanks! @drashna @noroadsleft

**Pull request merged**